src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,240 @@
+              40)
+  );
+
+  public static void main(String[] args) {
+    Logger.getLogger( "org.mongodb.driver" ).setLevel(Level.WARNING);
+    // TODO:
+    // Replace the placeholder connection string below with your
+    // Altas cluster specifics. Be sure it includes
+    // a valid username and password! Note that in a production environment,
+    // you do not want to store your password in plain-text here.
+    ConnectionString mongoUri = new ConnectionString("<Your Atlas Connection String>");
+
+    // Provide the name of the database and collection you want to use.
+    // If they don't already exist, the driver and Atlas will create them
+    // automatically when you first write data.
+    String dbName = "myDatabase";
+    String collectionName = "recipes";
+
+    // a CodecRegistry tells the Driver how to move data between Java POJOs (Plain Old Java Objects) and MongoDB documents
+    CodecRegistry pojoCodecRegistry = fromRegistries(MongoClientSettings.getDefaultCodecRegistry(),
+            fromProviders(PojoCodecProvider.builder().automatic(true).build()));
+
+    // The MongoClient defines the connection to our MongoDB datastore instance (Atlas) using MongoClientSettings
+    // You can create a MongoClientSettings with a Builder to configure codecRegistries, connection strings, and more
+    MongoClientSettings settings = MongoClientSettings.builder()
+            .codecRegistry(pojoCodecRegistry)
+            .applyConnectionString(mongoUri).build();
+
+    MongoClient mongoClient;
+    try {
+       mongoClient = MongoClients.create(settings);
+    } catch (MongoException me) {
+      System.err.println("Unable to connect to the MongoDB instance due to an error: " + me);
+      return;
+    }
+
+    // MongoDatabase defines a connection to a specific MongoDB database
+    MongoDatabase database = mongoClient.getDatabase(dbName);
+    // MongoCollection defines a connection to a specific collection of documents in a specific database
+    MongoCollection<Recipe> collection = database.getCollection(collectionName, Recipe.class);
+
+    /*      *** INSERT DOCUMENTS ***
+     *
+     * You can insert individual documents using collection.insert().
+     * In this example, we're going to create 4 documents and then
+     * insert them all in one call with insertMany().
+     */
+
+    try {
+      // recipes is a static variable defined above
+      InsertManyResult result = collection.insertMany(recipes);
+      System.out.println("Inserted " + result.getInsertedIds().size() + " documents.");
+    } catch (MongoException me) {
+      System.err.println("Unable to insert any recipes into MongoDB due to an error: " + me);
+      return;
+    }
+
+    /*      *** FIND DOCUMENTS ***
+     *
+     * Now that we have data in Atlas, we can read it. To retrieve all of
+     * the data in a collection, we call find() with an empty filter. find()
```

Comment:
`find()` doesn't return a cursor, which I think is apparent in your call to `iterator()`. See:
https://github.com/mongodb/docs-java/blob/master/source/usage-examples/find.txt

Perhaps you can avoid the verbose explanation and say something like this:
"We can retrieve an iterator to retrieve the results from our call to the `find()` method."